ui/lib/apps/Statement/utils/tableColumns.tsx

```diff
@@ -203,26 +224,56 @@ export function statementColumns(
 
     tcf.textWithTooltip('plan_digest', rows),
 
-    {
-      ...tcf.textWithTooltip('related_schemas', rows),
+    tcf.textWithTooltip('related_schemas', rows).patchConfig({
       minWidth: 160,
       maxWidth: 240,
-    },
-  ]
+    }),
+
+    // rocksdb
+    avgMaxColumn(
+      tcf,
+      'rocksdb_delete_skipped_count',
+      'short',
+      rows
+    ).patchConfig({
+      minWidth: 220,
+      maxWidth: 250,
+    }),
+    avgMaxColumn(tcf, 'rocksdb_key_skipped_count', 'short', rows).patchConfig({
+      minWidth: 220,
+      maxWidth: 250,
+    }),
+    avgMaxColumn(
+      tcf,
+      'rocksdb_block_cache_hit_count',
+      'short',
+      rows
+    ).patchConfig({
+      minWidth: 220,
+      maxWidth: 250,
+    }),
+    avgMaxColumn(tcf, 'rocksdb_block_read_count', 'short', rows).patchConfig({
+      minWidth: 220,
+      maxWidth: 250,
+    }),
+    avgMaxColumn(tcf, 'rocksdb_block_read_byte', 'short', rows).patchConfig({
```

Comment:
Some columns are using `tcf.xxx`, and some are using `xxx(tcf)`. This deserves some refactoring work later.

/cc @baurine I suggest you to give some design doc for this one as a guide for freshman.

Reply:
sorry, just saw this, ok, will add it later.